Library/Tracking/KSRAnalyticsTests.swift

```diff
@@ -734,22 +734,21 @@ final class KSRAnalyticsTests: TestCase {
     self.assertProjectProperties(segmentClient.properties.last)
   }
 
-  func testTrackCheckoutPaymentMethodViewed() {
+  func testTrackCheckoutPaymentMethodViewed_PledgeViewContext_Pledge() {
```

Comment:
So there's a similar test inside `testContextLocationProperties()`, maybe we delete them in favour on the methods you created?

Reply:
Ah I see. Pushed up a commit with the updates.